lib/pages/views/home.dart

```diff
@@ -150,44 +134,77 @@ class _HomePageState extends State<HomePage> {
           ),
           body: Column(
             children: <Widget>[
-              HomeScreenTopPart(),
-              HomeScreenBottomPart(),
-            ],
+            HomeScreenTopPart(),
+            FutureBuilder<List<Event>>(
+              future: EventController().getEvents(widget.user.token),
+              builder: (BuildContext context, AsyncSnapshot snapshot) {
+                if (!snapshot.hasData) {
+                  return Center(child: CircularProgressIndicator());
+                } else {
+                  eventList = snapshot.data;
+                  for (Event event in eventList) {
+                    String image = DotEnv().env['FATAPP_API'] + 'files/' + event.banner;
```

Comment:
Isso aqui não deveria estar incluso no model do evento? Quando pedir para transformar de JSON para o model, ele precisa fazer isso... Ai você evita esse laço `for`